foundry.toml

```diff
@@ -30,4 +30,10 @@ mainnet = "${ETHEREUM_RPC_URL}"
 goerli = "${GOERLI_RPC_URL}"
 polygon = "${POLYGON_RPC_URL}"
 
+
+[invariant]
+runs = 400
+depth = 15
+fail_on_revert = false
```

Comment:
runs is the number of times a randomized call sequence will be generated
depth is the number of random calls that will be made for a given run
fail_on_revert = false means that we expect some of the randomized calls to revert, and that's ok as long as our defined invariants hold in all cases

Reply:
My hunch is we'll want more depth to really push on our assumptions since doing anything useful with the market requires ~5 steps.